src/asteroid_dynamical_models.jl

```diff
@@ -41,300 +67,462 @@ end
+the eight planets, the Moon and Ceres, as well as the asteroid of interest as a test particle
+with null mass. Dynamical effects considered are:
+
+- Post-Newtonian point-mass accelerations between all bodies: see equation (35) in page 7 of https://ui.adsabs.harvard.edu/abs/1971mfdo.book.....M/abstract.
+
+- Figure-effects (oblateness) of the Earth (``J_2`` and ``J_3``),
+
+- ``J_2`` effect of the Sun and
+
+- ``J_2`` and ``J_3`` effect of the Moon: see equation (28) in page 13 of https://ui.adsabs.harvard.edu/abs/2014IPNPR.196C...1F%2F/abstract and equations (173) and (174) in page 33 of https://ui.adsabs.harvard.edu/abs/1971mfdo.book.....M/abstract.
+
+- Kinematic model for the precession and nutation of the Earth's orientation (IAU 1976/1980 Earth orientation model): see [`PlanetaryEphemeris.c2t_jpl_de430`](@ref).
+
+- Kinematic model for the Moon's orientation (Seidelmann et al., 2006): see equations (14)-(15) in page 9 and equations (34)-(35) in page 16 of https://ui.adsabs.harvard.edu/abs/2014IPNPR.196C...1F%2F/abstract.
+
+- Non-gravitational accelerations acting upon the asteroid: are included (Yarkovsky effect) 
+```math
+\mathbf{a}_\text{nongrav} = A_2\left(\frac{1 \ \text{au}}{r}\right)^d*\hat{\mathbf{t}},
+```
+where ``\hat{\mathbf{t}}`` is the unit heliocentric transverse vector, ``r`` is the
+asteroid's heliocentric range, ``A_2`` is a coefficient (with units of au/day^2),
+and ``d = 2``.
+
+See also [`PlanetaryEphemeris.NBP_pN_A_J23E_J23M_J2S!`](@ref).
+
+""" RNp1BP_pN_A_J23E_J2S_ng_eph!
+
 @taylorize function RNp1BP_pN_A_J23E_J2S_ng_eph!(dq, q, params, t)
-    local jd0 = params[4] # Julian date of start time
-    local dsj2k = t+(jd0-JD_J2000) # days since J2000.0 = 2.451545e6
-    local ss16asteph_t = evaleph(params[1], dsj2k, q[1]) # params[2](t)*one(q[1]) #ss16asteph(t)
-    local acceph_t = evaleph(params[2], dsj2k, q[1]) # params[2](t)*one(q[1]) #acc_eph(t)
-    local newtonianNb_Potential_t = evaleph(params[3], dsj2k, q[1]) # params[3](t)*one(q[1]) #newtonianNb_Potential(t), massive bodies
+    # Julian date of start time
+    local jd0 = params[4] 
+    # Days since J2000.0 = 2.451545e6
+    local dsj2k = t+(jd0-JD_J2000) 
+    # params[1](t)*one(q[1]) # ss16asteph(t)
+    local ss16asteph_t = evaleph(params[1], dsj2k, q[1]) 
+    # params[2](t)*one(q[1]) # acc_eph(t)
+    local acceph_t = evaleph(params[2], dsj2k, q[1]) 
+    # params[3](t)*one(q[1]) # newtonianNb_Potential(t), massive bodies
+    local newtonianNb_Potential_t = evaleph(params[3], dsj2k, q[1]) 
+    # Type of position / velocity components
     local S = eltype(q)
-    local UJ_interaction = params[5] # interaction matrix with flattened bodies
-    local N = params[6] # number of bodies, including NEA
+    # Interaction matrix with flattened bodies
+    local UJ_interaction = params[5] 
+    # Number of bodies, including NEA
+    local N = params[6] 
+    # Number of bodies, except Apophis
```

Comment:
Instead of "Apophis" I think it is better to write "asteroid" or "NEO"; the first suggestion is used in other files. Note that the same comment applies below and to other files.

Reply:
I've changed apophis for asteroid mostly in `src/asteroid_dynamical_models.jl`. In my opinion all other occurrences are apophis-specific.